pkg/security/secl/compiler/eval/context.go

```diff
@@ -37,6 +37,8 @@ type Context struct {
 	CachedAncestorsCount int
 
 	resolvedFields []string
+
+	Errored bool
```

Comment:
could we use en error object instead ? so that we could reuse the error field outside of this use case ? Also I would suggest to reset it in the Reset function ?

Reply:
yes and yes, let me update this